server/app.py

```
@@ -35,6 +35,7 @@ def create_app(config_name="development"):
     else:
         app.config.from_prefixed_env()
 
+    print
```

Ruff (0.13.1)

Found useless expression. Either assign it to a variable or remove it.

(B018)